test/MockConsole.cs

```diff
@@ -300,6 +300,17 @@ public virtual void BlankRestOfLine()
                 buffer[writePos + i].ForegroundColor = ForegroundColor;
             }
         }
+        
+        public virtual void BlankRestOfBuffer()
+        {
+            var writePos = CursorTop * BufferWidth + CursorLeft;
+            for (int i = 0; i < buffer.Length - writePos; i++)
```

Comment:
This new method can be simplified like the following:
```
for (; writePos < buffer.Length; writePos++)
{
    buffer[writePos].UnicodeChar = ' ';
    ...
}
```

Reply:
I think keeping it to the same pattern as `BlankRestOfLine()` et. al. makes more sense, but sure.